package.json

```diff
@@ -35,6 +35,7 @@
     "html-minifier-terser": "^7.2.0",
     "rehype-stringify": "^10.0.0",
     "remark-gfm": "^4.0.0",
+    "remark-html": "^16.0.1",
```

Comment:
I don't see it being used in your changes. Any reason to add it?

Reply:
https://github.com/nodejs/api-docs-tooling/blob/1e27644ca3daea1cca7d9a6b4bd7f4ae0b44627c/src/generators/legacy-json/utils/buildSection.mjs#L450-L457